doc/guide/sample.hpp

```diff
@@ -16,6 +16,23 @@ e.g., @c make @c mlpack_knn or @c make @c mlpack_test; see @ref build.  However,
 any program that uses mlpack (and is not a part of the library itself) can be
 compiled easily with g++ or clang from the command line.
 
+@section compile_link_and_run Compile, Link and Run C++ Files 
+
+Typically the following commands would be used to compile and run the C++ files.
+
+@code{.sh}
+g++ -std=c++11 filename.cpp -o filename
```

Comment:
Not sure we should provide that command line, since in most cases we have to link against mlpack, so I think the one you mentioned below should be the default. Let me know what you think.

Reply:
Yes,, i'll remove this part.

Reply:
If there is a way to compile and run using the above command line, please let me know.  (Some automatic linking or some environment variable could be used to look for -lmlpack)

From your comment, it seems clear that there might be no such "easy" way as mentioned above. So I would remove this command line. :)

Reply:
Yeah, I think this is a good starting point, and does improve what we had before so, from my side let's remove the first one and go with the one below.